consensus/src/aggregation/mocks/reporter.rs

```diff
@@ -63,7 +60,7 @@ impl<V: Variant, D: Digest> Reporter<V, D> {
     ) -> (Self, Mailbox<V, D>) {
         let (sender, receiver) = mpsc::channel(1024);
         let identity = *poly::public::<V>(&polynomial);
```

Comment:
Should we similarly define `public()` on `polynomial` like we did for `evaluate_all`?

Reply:
In the math refactor I've been working on, I've replaced this function with a method on Poly, so that could be nice, but I don't think we need to change that here.